espGarden/main/include/heltec_unofficial.h

```diff
@@ -237,15 +237,16 @@ void init_display() {
     }
 }
 
-void init_wifi(){
+void init_wifi() {
     ESP_ERROR_CHECK(wifi_manager_init());
     ESP_ERROR_CHECK(wifi_set_config(WIFI_SSID, WIFI_PASS));
     ESP_ERROR_CHECK(wifi_start());
 
-    // Print out connection status on display after connected and
-    // create a task to report the connection status in console
-    xTaskCreate(&wifi_print_stats, "wifi_stats", 4096, NULL, 5, NULL);
-}
+    while (!wifi_manager_is_connected()) {
```

Comment:
Is the idea to have no more wifi stats task and have the important info on the display? If so probably a good idea wifi logging gets annoying and removes another task we don't need

Reply:
This was before I saw log_stats()